### DISCUSSION TITLE
I build a centos image by compiling the source code on mac m1 aarch 64, but the cmake compilation fails

### DISCUSSION THREAD

**Discussion by ltdo111** [triton-inference-server/server#5488] [Q&A] [answered]
1. the occured error: 
![image](https://user-images.githubusercontent.com/17394163/224329988-d6b772e0-9ea4-457e-a0e8-d434f289ff94.png)

2. blow is my dockerfile 

ARG TRITON_VERSION=2.32.0dev
ARG TRITON_CONTAINER_VERSION=23.03dev
FROM  reg.docker.alibaba-inc.com/alipay/alios7u2-min:1.0-8eab639-20171023 as build-stage-0
USER admin
RUN sudo yum makecache fast
RUN sudo yum install -y epel-release \
    re \
    re2-devel \
    centos-release-scl-rh \
    git \
    vim \
    autoconf \
    automake \
    boost-devel \
    make \
    openssl-devel \
    python3-devel \
    wget \
    gcc gcc-c++ ||true
RUN sudo yum clean all
RUN sudo mkdir -p /workspace
RUN sudo chown -R admin:admin /workspace/
FROM build-stage-0 as build-stage-1
WORKDIR /workspace
RUN df -h
RUN sudo wget https://download-ib01.fedoraproject.org/pub/epel/7/x86_64/Packages/r/rapidjson-devel-1.1.0-2.el7.noarch.rpm && sudo yum install -y rapidjson-devel-1.1.0-2.el7.noarch.rpm
RUN sudo wget http://yum.oracle.com/repo/OracleLinux/OL8/developer/EPEL/x86_64/getPackage/libb64-devel-1.2-6.el8.x86_64.rpm &&  sudo yum install -y libb64-devel-1.2-6.el8.x86_64.rpm
RUN curl http://alipay-cognition.cn-hangzhou.alipay.aliyun-inc.com/datacube2%2Fpackages%2Flibrary%2Fcmake.tar.gz -o cmake.tar.gz && sudo tar zxvf cmake.tar.gz && sudo rm -rf cmake.tar.gz
RUN cd cmake-3.19.1 && sudo ./bootstrap && sudo make install -j16
RUN sudo rm -rf cmake-3.19.1
COPY aci/server ./
RUN ls -la && cd build && sudo ./cmake_build
ENTRYPOINT []

4. my project blow : 

![image](https://user-images.githubusercontent.com/17394163/224329814-591fba28-cabd-41d7-9510-eeab72d71814.png)

**Comment by ltdo111** [comment]
cmake-3.19.1

**Comment by the-david-oy** [comment, marked as answer]
We don't officially support this architecture, so we can't provide much guidance. Also, the error likely isn't where you posted. The error should be higher up in the logs. If you're running `make` with multiple threads, you may want to try doing it single-threaded (e.g. not `-j16` or other j flags/values).
